relay_test.go

```diff
@@ -71,7 +71,6 @@ func DisabledTestRelayHandlesCrashedPeers(t *testing.T) {
 }
 
 func TestRelayConnectionCloseDrainsRelayItems(t *testing.T) {
-	opts := serviceNameOpts("s1").SetRelayOnly()
```

Comment:
whoops, was doing testing with runcount set to 1000, and "reverted" by just deleting the whole line. travis is unhappy.